jdaviz/app.py

```diff
@@ -790,6 +791,8 @@ def _link_new_data(self, reference_data=None, data_to_be_linked=None):
         if self.config == 'mosviz':
             # In Mosviz, first data is always MOS Table. Use the next data
             default_refdata_index = 1
+        elif self.config == 'cubeviz':
```

Comment:
I would love for this to do something other than check the config (thinking of deconfigging efforts), but everything else in this block follows the same pattern, so probably makes sense to defer that